demo/atom/switch/playground

```diff
@@ -43,6 +49,15 @@ return (
     </div>
```

Comment:
I suggest removing "Disabled" from this section, as it's already present in the next section